yw_website/yw_website/apps/website/models.py

```diff
@@ -58,7 +58,7 @@ class Run(models.Model):
 
 class File(models.Model):
     checksum = models.CharField(
-        max_length=128, primary_key=True)
```

Comment:
this is a good change, it looks like even if a files contents are identical, they can still generate different hashes. we should have looked into this earlier :/